yew/src/html/scope.rs

```diff
@@ -236,6 +257,8 @@ struct ComponentState<COMP: Component> {
 }
 
 impl<COMP: Component> ComponentState<COMP> {
+    /// Creates a new `ComponentState`, also invokes the `create()`
+    /// method on component to create it.
```

Comment:
nit: internal structs don't need "doc" comments. Normal comments are fine here (`///` -> `//`)

Reply:
it doesn't hurt though or it's part of the code style guidelines for yew? Happy to change if required, I prefer them because they show up in IDE documentation popups when working with this code, and if it's ever made public (not this item specifically, but any other internal item), it's closer to being ready.

Reply:
Oh ok, didn't realize they helped in the IDE. Sounds good!